src/main/kotlin/Main.kt

```diff
@@ -25,59 +27,80 @@ fun main() {
             MenuState.CREATE -> {
                 when {
                     selectedArchiveIndex < 0 -> {
-                        println("Введите наазвание архива")
-                        if (scanner.hasNextLine()) {
+                        while (true) {
+                            println("Введите наазвание архива")
```

Comment:
🍏 Закралась опечатка:
```suggestion
                            println("Введите название архива")
```